model/tensorflow_hub/dffml_model_tensorflow_hub/text_classifier_accuracy.py

```diff
@@ -24,19 +25,17 @@ class TextClassifierAccuracyContext(AccuracyContext):
     as test data.
     """
 
-    async def score(
-        self, records: AsyncIterator[Record], features: List[Feature],
-    ):
+    async def score(self, modelcontext: ModelContext, sources: Sources):
         if not os.path.isfile(
-            os.path.join(self.model_dir_path, "saved_model.pb")
+            os.path.join(modelcontext.model_dir_path, "saved_model.pb")
         ):
             raise ModelNotTrained("Train model before assessing for accuracy.")
-        x, y = await self.train_data_generator(records)
-        accuracy_score = self._model.evaluate(x, y)
+        x, y = await modelcontext.train_data_generator(sources)
+        accuracy_score = modelcontext._model.evaluate(x, y)
         return accuracy_score[1]
```

Comment:
The implementation of this makes me think that we might want to remove the accuracy method from the models entirely. We should talk more about this in the meeting, and we may not necessarily need to do it within phase 4, it could wait.

I say this because we seem to have a few cases now where the accuracy scorer might needs more information from / about the model. But the only case we have where the model needs more information about the accuracy scorer is when we're saying that a certain accuracy scorer won't work or won't be used.